docs/RouteRequest.md

```diff
@@ -192,22 +192,34 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 **Since version:** `2.0` ∙ **Type:** `duration` ∙ **Cardinality:** `Optional` ∙ **Default value:** `"PT0S"`   
 **Path:** /routingDefaults 
 
-The minimum extra time after exiting a public transport vehicle.
+The time safety margin when alighting from a vehicle.
+
+This time slack is added to arrival time of the vehicle before any transfer or onward travel.
+
+The sole reason for this is to avoid missed connections when there are minor schedule variations. This
```

Comment:
I wasn't properly concentrating today during the developer meeting discussion, but isn't this the description of transferSlack, not alightSlack? I thought alightSlack was mostly meant just for random procedures you have to do after you alight from a vehicle like with airplanes. Although, this is the generic one and I'm not even completely sure what this is for. Maybe if you are traveling on wheelchair this could be useful you if you need assistance or if the vehicles are crowded and getting off a vehicle takes time.

Referring to connection feels a bit weird. This feels more like a safety margin that you get to the destination in time.

Reply:
I agree with you a wrote something to that effect previously but @abyrd and @t2gran dictated this to me word for word.

I'm feeling that I should just revert the change to the documentation and we pick that up later.

Reply:
It's possible that I misinterpreted the purpose of the different parameters. I tried to verify with everyone that any strong statements were accurate. I'm happy to rewrite it if it's incorrect.

Reply:
To me the description is good. The following sentence could be better:

> The sole reason for this is to avoid missed connections when there are minor schedule variations.

It should emphasise "minor schedule variations". Note! It is probably a better simulation of real life to have 20s boardSlack and 10s alightSlack, than 30s/0s. But, due to technical difficulties in presenting this to the end user, most deployments uses just boardSlack. I do not recommend using this for user-specific needs like wheelchair.

We can refrase to, if less confusing:

> The main reason is to account for minor schedule variations.